### PR TITLE
[18.0][MIG] partner_identification_unique_by_category: Migration to 18.0

### PR DESCRIPTION
**Pull request by chaule97**

### CODE REVIEW

File: partner_identification_unique_by_category/tests/test_partner_identification_unique_by_category.py [hunk 1]
```diff
@@ -0,0 +1,84 @@
+# Copyright 2021 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+from odoo.exceptions import ValidationError
+
+from odoo.addons.base.tests.common import SavepointCaseWithUserDemo
```
Comment (TDu) [inline partner_identification_unique_by_category/tests/test_partner_identification_unique_by_category.py]:
```suggestion
from odoo.tests.common import TransactionCase
```

Comment (chaule97) [reply partner_identification_unique_by_category/tests/test_partner_identification_unique_by_category.py]: Hi @TDu , I have updated it

Comment (TDu) [reply partner_identification_unique_by_category/tests/test_partner_identification_unique_by_category.py]: Thanks

**Review by TDu — APPROVED** [review]

**Comment by rousseldenis** [comment]
@chaule97 Thanks for this. Don't forget to add the depending PR in this one description to help review

**Comment by chaule97** [comment]
Salut @rousseldenis , I have rebase. Could you merge again?

**Comment by rousseldenis** [comment]
> Salut @rousseldenis , I have rebase. Could you merge again?

`migration`command is not a merge. Just to update Migration PR :sweat_smile:

File: partner_identification_unique_by_category/tests/test_partner_identification_unique_by_category.py (hunk 1 above)
Comment (simahawk) [reply partner_identification_unique_by_category/tests/test_partner_identification_unique_by_category.py]: actually there's a better one that takes care of turning off tracking and other flags: `from odoo.addons.base.tests.common import BaseCommon`

File: partner_identification_unique_by_category/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+    "name": "Partner Identification Numbers Unique By Category",
+    "category": "Customer Relationship Management",
+    "version": "18.0.1.0.0",
+    "license": "AGPL-3",
+    "depends": ["partner_identification"],
+    "data": ["views/res_partner_id_category_view.xml"],
+    "author": "Camptocamp SA, Odoo Community Association (OCA)",
```
Comment (simahawk) [inline partner_identification_unique_by_category/__manifest__.py]:
```suggestion
    "author": "Camptocamp, Odoo Community Association (OCA)",
```

**Review by simahawk — APPROVED** [review]
Minor remark for author name

**Pull request merged**